nck/writers/objectstorage_writer.py

```diff
@@ -0,0 +1,78 @@
+# GNU Lesser General Public License v3.0 only
+# Copyright (C) 2020 Artefact
+# licence-information@artefact.com
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+import os
+
+from nck.config import logger
+from nck.writers import Writer
+from nck.utils.retry import retry
+
+
+class ObjectStorageWriter(Writer):
+    def __init__(self, bucket_name, prefix=None, file_name=None, platform=None, **kwargs):
+        self._bucket_name = bucket_name
+        self._prefix = prefix if prefix else ""
+        self._file_name = file_name
+        self._platform = platform
+        self._bucket = self._get_bucket_if_exist()
+
+    @retry
+    def write(self, stream):
+        logger.info(f"Start writing file to {self._platform} ...")
+        print(self._file_name)
```

Comment:
It is generally advised to use the `logging` package instead of `print` statements

Reply:
sorry these are relicas from when i was debugging